modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/search/engine/adapter/search/ClearScrollRequestExecutorImpl.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.search.elasticsearch7.internal.search.engine.adapter.search;
+
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.search.elasticsearch7.internal.connection.ElasticsearchClientResolver;
+import com.liferay.portal.search.engine.adapter.search.ClearScrollRequest;
+import com.liferay.portal.search.engine.adapter.search.ClearScrollResponse;
+
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestHighLevelClient;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Gustavo Lima
+ */
+@Component(service = ClearScrollRequestExecutor.class)
+public class ClearScrollRequestExecutorImpl
+	implements ClearScrollRequestExecutor {
+
+	@Override
+	public ClearScrollResponse execute(ClearScrollRequest clearScrollRequest) {
+		org.elasticsearch.action.search.ClearScrollRequest
+			elasticsearchClearScrollRequest =
+				new org.elasticsearch.action.search.ClearScrollRequest();
```

Comment:
i typically put the request instantiation inside its own method, even if its trivial. not a big deal, but thats generally the pattern ive used, create*Request and get*Response methods inside the *ExecutorImpl classes 

see FollowInfoCCRRequestExecutorImpl#_createFollowInfoRequest as an example